Ardupilog.m

```diff
@@ -6,7 +6,7 @@
 % This is hard-coded:
 % - The 128 (FMT) message must have fields "Name", "Type", and "Length" which specify other LogMsgGroups
 
-classdef Ardupilog < dynamicprops
+classdef Ardupilog < dynamicprops & matlab.mixin.Copyable
```

Comment:
This inherits handle functionality (similar to C++ pointers). Work is done on the same object insitu.